packages/vite/src/node/plugins/resolve.ts

```diff
@@ -690,7 +690,7 @@ export function tryNodeResolve(
     // otherwise we may introduce duplicated modules for externalized files
     // from pre-bundled deps.
     if (!isBuild) {
-      const versionHash = depsOptimizer.metadata({ ssr }).browserHash
+      const versionHash = depsOptimizer.metadata({ ssr })?.browserHash
```

Comment:
This is always defined, why was this change needed?

Reply:
It isn't defined when the module is CSS (tried with `sanitize.css`) maybe because it can't be optimized? This is new by the way, the previous version didn't need this.

Reply:
I tried this repro https://github.com/cyco130/vite-ssr-react/tree/ssr-css-bug, against beta.5 and it is working 🤔
Could you create a failing repro so I can check what is going on here?

Reply:
I found why you need this, I'll send a PR tomorrow to fix it. Still a repro of the CSS issue against latest would be good

Reply:
> Could you create a failing repro so I can check what is going on here?

I updated the repro now and it is failing for me. Maybe try `rm -rf node_modules/.vite`?